<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="45729" started-at="2016-12-25T19:15:52Z" finished-at="2016-12-25T19:16:38Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="45729" started-at="2016-12-25T19:15:52Z" finished-at="2016-12-25T19:16:38Z">
      <class name="socialtrade.bin.TodayTaskTest">
        <test-method status="PASS" signature="testLogin()[pri:0, instance:socialtrade.bin.TodayTaskTest@334cc8ce]" name="testLogin" is-config="true" duration-ms="13066" started-at="2016-12-26T00:45:52Z" finished-at="2016-12-26T00:46:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin -->
        <test-method status="FAIL" signature="testStartWorking()[pri:0, instance:socialtrade.bin.TodayTaskTest@334cc8ce]" name="testStartWorking" duration-ms="32440" started-at="2016-12-26T00:46:05Z" finished-at="2016-12-26T00:46:38Z">
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: null
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertTrue(Assert.java:27)
	at socialtrade.bin.TodayTaskTest.testStartWorking(TodayTaskTest.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testStartWorking -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:socialtrade.bin.TodayTaskTest@334cc8ce]" name="tearDown" is-config="true" duration-ms="160" started-at="2016-12-26T00:46:38Z" finished-at="2016-12-26T00:46:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- socialtrade.bin.TodayTaskTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
